plugins/legacy-plugin-chart-force-directed/src/ReactForceDirected.jsx

```diff
@@ -16,7 +16,44 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import React from 'react';
 import { reactify } from '@superset-ui/chart';
+import styled from '@superset-ui/style';
+import PropTypes from 'prop-types';
```

Comment:
Since this component is relatively simple, maybe also migrate to TypeScript and get rid of PropTypes?

Reply:
If i do this there's an issue where it doesn't recognize the import ./ForceDirected